utils/forest_shim/src/clock.rs

```diff
@@ -0,0 +1,8 @@
+// Copyright 2019-2023 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0, MIT
+
+/// Duration of each tipset epoch.
+pub const EPOCH_DURATION_SECONDS: i64 = 30;
```

Comment:
why do we redefine this? It should be available in the FVM.

Reply:
I vote use what we can from the FVM but don't use anything from the actor runtimes.

Reply:
@lemmih why? We are using the entire `Policy` struct from [here](https://github.com/filecoin-project/builtin-actors/blob/master/runtime/src/runtime/policy.rs)

Reply:
Indeed, and that has been a nightmare. A new copy of the `Policy` is pushed every network upgrade. We need a proper way of dealing with that. Right now, we're just kicking the can down the road.